thirdparty_containers/system-test/playlist.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>system-test</testCaseName>
+		<variations>
+			<variation>-XX:+UseContainerSupport</variation>
+		</variations>
+		<command>docker run -v $(JDK_HOME):/java --name system-test adoptopenjdk-system-test:latest ; \
+		 docker cp system-test:/tmp/stf  $(REPORTDIR)/external_test_reports; \
```

Comment:
Is the content of external_test_reports all of the stf output plus the generated .tap file?

Reply:
I am not sure about what the content of /tmp/stf here would be. System tests do not generate any junit xml report like the application tests. So the `docker cp` command here can be removed for the time being. When /if we update system tests to generate reports, we can have this step added back in.